client/components/comment.jsx

```diff
@@ -80,6 +108,11 @@ export default class Comment extends React.Component {
     }
 
     render() {
+        const likeStatus = this.state.like ? 'Liked' : 'Like';
```

Comment:
```suggestion
        const { like,  repost, share, show } = this.state
        const likeStatus = like ? 'Liked' : 'Like';
```
Use object destructuring for this